.github/workflows/release-nightly.yml

```diff
@@ -40,3 +40,24 @@ jobs:
         run: yarn run publish:nightly
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Get latest tag
+        uses: actions-ecosystem/action-get-latest-tag@v1
+        with:
+          with_initial_version: false
+        id: tag
+
+  docker:
+    name: Publish to Docker Hub
+    runs-on: ubuntu-latest
+    needs: publish
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      # TODO: Prevent Docker Hub tagging this image as "latest", tag as "next" instead
```

Comment:
just add 
`docker tag chainsafe/lodestar:${{ needs.publish.outputs.tag }} chainsafe/lodestar:next`
and also push chainsafe/lodestar:next

Reply:
Thanks! What about preventing dockerhub from tagging it as latest?

Reply:
I don't think dockerhub does that by default. You have to push latest tag in order to update it or push image without tag